scripts/push-docker-images.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Push our docker images to a registry
+set -xeuo pipefail
+
+# Manually set the registry to a docker registry where you have push access to test this script.
+REGISTRY=${REGISTRY:-gomods/}
```

Comment:
is there a pros/cons over using value without `/`?  I feel like I will miss the forwarding `/` when I'm setting it manually

Reply:
It's the convention that I've seen other repositories use. This let's you set REGISTRY to "" and then the docker image is just "proxy:latest" which is nice for local work.

Once this is in, we can follow-up with documentation for how to build and use the docker image locally in our DEVELOPMENT.md doc, in addition to explaining what the various docker image tags mean.